spec/fixtures/native-addon/external-ab/binding.cc

```diff
@@ -1,5 +1,4 @@
 #include <node_api.h>
-#undef NAPI_VERSION
```

Comment:
This used to be necessary due to a conflict in `#include <node_buffer.h>` and `#include <node_api.h>` born of duplicate declarations, and isn't anymore as the duplicates got refactored out.